Sources/Sentry/SentryOptions.m

```diff
@@ -48,7 +48,7 @@ - (instancetype)init
         _defaultTracesSampleRate = nil;
         self.tracesSampleRate = _defaultTracesSampleRate;
 
-        // Use the name of the bundle’s executable file as inAppInclude, so SentryFrameInAppLogic
+        // Use the name of the bundle’s executable file as inAppInclude, so SentryInAppLogic
         // marks frames coming from there as inApp. With this approach, the SDK marks public
```

Comment:
So no more frames get marked?

Reply:
You still can use the class to check if a frame is in-app or not, but you don't do that by passing in the frame but instead the image name. See 

> // Use the name of the bundle’s executable file as inAppInclude, so SentryInAppLogic
> // marks frames coming from there as inApp. With this approach, the SDK marks public

Reply:
Oh, I see. That makes sense.